test/parallel/test-inspect-support-for-node_options.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const common = require('../common');
+const cluster = require('cluster');
+const assert = require('assert');
+
+if (process.config.variables.node_without_node_options)
+  common.skip('missing NODE_OPTIONS support');
+
```

Comment:
Could you also call `common.skipIfInspectorDisabled()`?

Reply:
Done